code/__HELPERS/stack_trace.dm

```diff
@@ -1,15 +1,32 @@
+// using datum is best, but "SHOULD_BE_PURE" lint doesn't like that way. list works well.
+GLOBAL_LIST_INIT(runtime_helper, list(
+	"runtime_message" = STACK_TRACE_NULL_HINT,
+	"file",
```

Comment:
These need to have assoc values, probably something like null

Reply:
ah yeah, good point